simularium_metrics_calculator/bin/example.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from simulariumio import InputFileData
+
+from simularium_metrics_calculator import (
+    METRIC_TYPE,
+    PLOT_TYPE,
+    SCATTER_PLOT_MODE,
+    MetricsManager,
+    PlotInfo,
+)
+
+
+def main() -> None:
+    # get the metrics that are available to plot
+    time_metrics = MetricsManager.available_metrics(METRIC_TYPE.PER_TIME)
+    print(f"\nAvailable per time metrics:\n{time_metrics}")
+    agent_metrics = MetricsManager.available_metrics(METRIC_TYPE.PER_AGENT)
+    print(f"\nAvailable per agent metrics:\n{agent_metrics}")
+
+    # configure some plots
+    plot1 = PlotInfo(  # Number of agents vs time scatterplot
+        plot_type=PLOT_TYPE.SCATTER,
+        metric_id_x=0,
+        metric_id_y=2,
+        scatter_plot_mode=SCATTER_PLOT_MODE.LINES,
+    )
+    plot2 = PlotInfo(  # Nearest neighbor distance histogram
+        plot_type=PLOT_TYPE.HISTOGRAM,
+        metric_id_x=3,
```

Comment:
Just to clarify for myself, generally we'd want to make sure that these IDs align with whatever we're getting back as available metrics from the call to `MatricsManager.available_metrics(...)` above, correct? Like in actual production code, we shouldn't just use any hard coded values because we can't assume that the metrics dictionary is constant?

Reply:
yes, the client should request metrics in order to populate UI and then use the IDs it got from `MetricsManager.available_metrics(...)` to request plots with those metrics